public/views/aeon/_aeon_request_action.html.erb

```diff
@@ -14,8 +14,10 @@ mapper = case record
   end
 %>
 
+<% unless mapper.hide_button? %>
+
 <% if mapper.show_action? %>
```

Comment:
Please indent the contents of the `unless mapper.hide_button?` block by 2 spaces.